docs/team_tasks.md

```diff
@@ -0,0 +1,61 @@
+## TEAM MEMBERS AND TASKS
+
+## Ekemini - Data Cleaning and Preprocessing:
```

Comment:
Great job assigning the task, can you please update the file to address members by their Github usernames?